wifiphisher/pywifiphisher.py

```diff
@@ -1023,8 +1036,10 @@ def run():
 
     clients_APs = []
     APs = []
-    args.accesspoint = ap_mac
-    args.channel = channel
+
+    if not (args.essid and args.channel):
+        args.accesspoint = ap_mac
```

Comment:
I'm also concerned that we are using the same variable name `ap_mac` for both the MAC retrieved from the target AP and our MAC address. Can you rename the last one to `our_mac`?